www/nginx/src/opnsense/www/js/nginx/src/templates/logviewer.html

```diff
@@ -1,93 +1,52 @@
+                </div>
+            </th>
+            <% }) %>
         </tr>
     </thead>
     <tbody>
     </tbody>
     <tfoot class="sticky-bottom">
         <tr>
-            <th class="text-nowrap active">
+            <th class="text-nowrap active" colspan="100">
+                <div class="btn-group dropup"><button type="button" class="btn btn-primary dropdown-toggle" data-toggle="dropdown" aria-expanded="false"><span class="dropdown-text"><span class="icon glyphicon glyphicon-th-list"></span></span> <span class="caret"></span></button>
+                   <ul class="dropdown-menu">
+                       <% log_fields.forEach( (field) => { %>
+                       <li><label class="ngx-dropdown-item"><input type="checkbox" class="option" value="<%= _.escape(field.id) %>" <%= field.visible === true ? 'checked="checked"' : '' %>/> <%= _.escape(field.header) %></label></li>
+                       <% }) %>
+                   </ul>
+                </div>
                 <button class="btn btn-primary" id="paging_first" title="First Page"><i class="fa fa-fast-backward" aria-hidden="true"></i></button>
                 <button class="btn btn-primary" id="paging_back" title="Previous Page"><i class="fa fa-step-backward" aria-hidden="true"></i></button>
                 <button class="btn btn-primary" id="refresh" title="Refresh"><i class="fa fa-refresh" aria-hidden="true"></i></button>
                 <button class="btn btn-primary" id="paging_forward" title="Next Page"><i class="fa fa-step-forward" aria-hidden="true"></i></button>
                 <button class="btn btn-primary" id="paging_last" title="Last Page"><i class="fa fa-fast-forward " aria-hidden="true"></i></button>
+                &nbsp;&nbsp;Page <span id="currentpage">1</span>/<span id="pagecount">0</span>
+                <div style="float: right; display: flex;">
+                    <label for="entrycount"><span id="entrycountdisplay">100</span> lines per page</label><input type="range" list="tickmarks" min="5" max="1000" step="5" id="entrycount" value="100" style="width: 200px;" />
+                    <datalist id="tickmarks">
+                        <option value="5" label="5">
+                        <option value="100" label="100">
+                        <option value="200" label="200">
+                        <option value="300">
+                        <option value="400">
+                        <option value="500" label="500">
+                        <option value="600">
+                        <option value="700">
+                        <option value="800">
+                        <option value="900">
+                        <option value="1000" label="1000">
+                    </datalist>
+                </div>
+                &nbsp;&nbsp;Found lines: <span id="resultcount">0</span>/<span id="totalcount">0</span>
```

Comment:
I would like to avoid abusing whitespace characters for layouting

Reply:
@fabianfrz thanks for review. please clarify, would you prefer just a `&ensp;` instead (here and at line 31) or use styles for this (I think it might be overkill ..)?

Reply:
@fabianfrz, did I understand you correctly? is it better now?

Reply:
no, no html entities if possible

Reply:
@fabianfrz ah! so its ok with padding now?)